src/components/AdminFields/Organizations/OrganizationPermissions/OrganizationEditor.js

```diff
@@ -108,8 +108,12 @@ class OrganizationEditor extends React.Component {
      */
     fetchAdminOptions = async () => {
         try {
-            const response = await client.get('organization_class')
-            return this.setState({classifications: response.data.data})
+            const {data} = (await client.get('organization_class'))?.data;
+            // make Muu yhteisö class = org:10 last choice
+            const otherClass = data.find((org) => org.id === 'org:10') ?? {},
```

Comment:
If `org:10` is missing for some reason, `{}` would be added to the result array. This might cause issues down the line when the result array is being used.

Reply:
fixed!

Reply:
Now if `org:10` is missing the result `sortedClasses` gets value undefined and causes a crash when dealing with the org options.

As a side note, you don't have to worry too much about the coverage test results when the new additions are not feasible to test with jest. I would say the istanbul ignores are not needed and coverage can be left to tell us the realistic situation of tests. Also, the force pushing makes it harder to keep track of what has happened. When we have the branch pulled locally to test it, we also have to manually resolve the force pushed content. Squash and merge can be used later to make the commit neat.

Reply:
Good that you caught it, that was a silly mistake :)